Fix README.md and add requirements.txt

davidyuk opened the pull request.

davidyuk committed:
README.md: Fix commands list markdown layout

davidyuk committed:
Use requirements.txt for project dependencies

FooBarrior commented:
It is not clear what kind of fix you made from a9e7098 commit message. Please, change it to smth like ' README.md: Fix commands list markdown layout'

FooBarrior commented on README.md:
```diff
@@ -4,8 +4,8 @@ CLI for [cats](http://imcs.dvfu.ru/cats). Currently only for jury
 - Install [python3](https://python.org)
 - Install [git](https://git-scm.com/)
-- Install [requests](http://docs.python-requests.org/en/latest/) python package with pip
 - Clone this repository
```
nice to explicitly specify `git clone` command too

I suggest You to change this part as following:
- Install [python3](https://python.org) and [git](https://git-scm.com/)
- Clone repo, install requirements, add cats.py to your PATH

``` bash
git clone https://github.com/FooBarrior/cats-cli
pip install -r requirements.txt
ln -s cats.py ~/.local/bin/cats
```

davidyuk committed:
README.md: More verbose installation guide

Restore removed @FooBarrior commit from
https://github.com/Davidyuk/cats-cli/pull/1

davidyuk commented:
Ok, I tried to fix all inaccuracies.
